sierra-previewer/src/main/java/org/httprpc/sierra/previewer/MainFrame.java

```diff
@@ -46,28 +45,21 @@
  * business logic.
  */
 public class MainFrame extends JFrame {
-
-	private static final long serialVersionUID = 2628641249775639871L;
-
 	// --- Subsystems ---
 	private final RenderingEngine renderingEngine;
 	private final Timer debounceTimer;
 	private final RecentFilesManager recentFilesManager; // NEW: Manager instance
 
 	// --- File Handling State ---
 	private final JFileChooser fileChooser;
-	private Path currentFilePath = null;
-	// --- UI Components (Injected by Sierra) ---
-	// Stores the path of the currently loaded file
 
-	@Outlet
-	private JMenuBar menuBar;
-
-	@Outlet
-	private JMenuItem openItem;
+    // Stores the path of the currently loaded file
+	private Path currentFilePath = null;
 
-	@Outlet
-	private JMenuItem saveItem;
+	// --- UI Components (Injected by Sierra) ---
```

Comment:
Current convention is to declare outlets as follows, although obviously either will work.